### PR TITLE
create app

### PR DESCRIPTION
**Pull request by khedkarpooja**

### CODE REVIEW

File: samplereact1/.gitignore [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```
Comment (visha15l) [inline samplereact1/.gitignore]: do not push gitignore file

File: samplereact1/src/App.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import './App.scss';
+import Main from './main';
+
```
Comment (visha15l) [inline samplereact1/src/App.js]: remove blank lines

File: samplereact1/src/App.scss [hunk 3]
```diff
@@ -0,0 +1,135 @@
+$font-p: 12px;
```
Comment (visha15l) [inline samplereact1/src/App.scss]: format scss file

File: samplereact1/src/main.js [hunk 4]
```diff
@@ -0,0 +1,78 @@
+import React, { Component,Fragment } from "react";
+
+
+
+class Main extends Component {
+  state = {
+    flu_drugs: [
```
Comment (visha15l) [inline samplereact1/src/main.js]: move this to constant file.

File: samplereact1/.gitignore (hunk 1 above)
Comment (khedkarpooja) [reply samplereact1/.gitignore]: ok

**Pull request merged**